resources/candlepin-lint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+
+env | sort
+echo
+
+docker pull docker-registry.usersys.redhat.com/candlepin/candlepin-postgresql || true
+echo "Docker pull completed"
+
+# The docker container test script will know to copy out
+echo "Using workspace: $WORKSPACE"
+mkdir -p $WORKSPACE/artifacts/
+
+# make selinux happy via http://stackoverflow.com/a/24334000
+chcon -Rt svirt_sandbox_file_t $WORKSPACE//artifacts/
+
+# Fix PR author if necessary
+if [ "$ghprbPullAuthorEmail" == "" ]; then
+  `git log -n 1 --format="export ghprbPullAuthorEmail=%aE"`
+fi
+
+echo "Using PR author e-mail: $ghprbPullAuthorEmail"
```

Comment:
Yeah, that destination line works. I was somewhat hopeful that with this change we had a means of detecting the PR/commit author with more reliability.